src/appkit.cs

```diff
@@ -1981,6 +2001,31 @@ public interface NSButton : NSAccessibilityButton {
 		[Export ("initWithFrame:")]
 		IntPtr Constructor (CGRect frameRect);
 
+		[Mac (10,12)]
+		[Static]
+		[Export ("buttonWithTitle:image:target:action:")]
+		NSButton ButtonWithTitle (string title, NSImage image, [NullAllowed] NSObject target, [NullAllowed] Selector action);
```

Comment:
* Missing verb for method name, e.g. `Get` (or `Create`) prefix;
* Extraneous `WithTitle` suffix, the first parameter makes that clear;
* Same applies for the next overloads